### PR TITLE
Update typography section

### PR DESCRIPTION
**Pull request by vpereira**
I Added decision to go with the native stack as default. As discussed before, I removed some uncessary examples of bootstrap usage, which aren't really used in OBS.

### CODE REVIEW

**Comment by hennevogel** [comment]
Deploy preview for *obs-patterns* ready!

Built with commit 61e6c0406f36d84104aefaf49488f613fde44685

https://deploy-preview-63--obs-patterns.netlify.com

File: _posts/atoms/2018-09-27-typography.html [hunk 1]
```diff
@@ -2,41 +2,12 @@
...
-%p.h4 Paragraph with .h4
-%p.h5 Paragraph with .h5
-%p.h6 Paragraph with .h6
-
-%h1
-  h1 Heading 1
-  %small.text-muted with faded text
 
-%hr
-%h5 Inline Text Elements
```
Comment (dmarcoux) [inline _posts/atoms/2018-09-27-typography.html]: We should keep this as there are still other inline text elements even though `%mark` is now removed.

File: _posts/atoms/2018-09-27-typography.html [hunk 2]
```diff
@@ -2,41 +2,12 @@
 layout: post
 title: Typography
 category: atoms
-description: We stick to [Bootstrap's global settings for typography](https://getbootstrap.com/docs/4.1/content/typography/#global-settings).
-             We use Bootstrap [typography](https://getbootstrap.com/docs/4.1/content/typography/) and [text](https://getbootstrap.com/docs/4.1/utilities/text/) classes when needed.
+description: After some research we decided to stick with system fonts as well known as [Native font stack](https://getbootstrap.com/docs/4.1/content/reboot/#native-font-stack).
```
Comment (dmarcoux) [inline _posts/atoms/2018-09-27-typography.html]: Missing comma between `research` and `we`.

The part `with system fonts as well known as Native font stack` is a bit weird. I suggest something like:

After some research, we decided to stick with the [Native font stack](https://getbootstrap.com/docs/4.1/content/reboot/#native-font-stack) as we want to rely on system fonts.

**Review by dmarcoux — CHANGES_REQUESTED** [review]

File: _posts/atoms/2018-09-27-typography.html (hunk 2 above)
Comment (vpereira) [reply _posts/atoms/2018-09-27-typography.html]: updated

**Review by dmarcoux — APPROVED** [review]

**Pull request merged**